pkgs/development/tools/misc/rtss/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rtss";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "Freaky";
+    repo = pname;
+    rev = "dfbf8d060c82b4c55f5366a3654ae4ca398ede7c";
+    sha256 = "1ij10kb4162q9rfwrflp3h9g2ldnxyd1qvjl5zdf1yagwijd1363";
+  };
+
+  doCheck = false;
```

Comment:
Can you comment why you added this?

Reply:
I noticed that the code mentioned in this error was commented out upstream, and assumed it wasn't used. It was actually a failing doctest that was unfixed upstream.

I fixed it: https://github.com/Freaky/rtss/pull/5

I requested another release as well - that way I can change the `src` attribute in this package and remove `doCheck = false`.

I think the test failure might have been a typo:
```diff
commit 5e85f729fe4e6382b81ae07a268b51c7300c353f (HEAD -> master, djanatyn/master)
Author: Jonathan Strickland <djanatyn@gmail.com>
Date:   Sat Nov 28 18:25:38 2020 -0500

    Fix failing test E0308

diff --git a/src/lib.rs b/src/lib.rs
index 0cccbd5..04d97fd 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -112,7 +112,7 @@ impl<W: io::Write> RtssWriter<W> {
     /// use rtss::{RtssWriter, DurationExt};
     ///
     /// fn main() -> io::Result<()> {
-    ///     let mut writer = RtssWriter::new(io::stdout(), Duration::human_string, '|', &Instant::now());
+    ///     let mut writer = RtssWriter::new(io::stdout(), Duration::human_string, '|', Instant::now());
     ///     writer.write(b"Hello!\n")?;
     ///     writer.write(b"World!\n")?;
     ///     Ok(())
```

Reply:
The PR was merged upstream and a new release was issued.

I pushed 4004ed9 to:
1. Use the new release, pulling `v0.6.1` tag as `src`
2. Remove `doCheck = false`